### ISSUE TITLE
Compatiblity

### ISSUE THREAD

**Issue by R-Miller** [ithinc/tabutils#6]
I attempted to install version 26 with an update of Tab Utilities 1.5.1 to 1.5.20 that was suppose to be compatible with version 26. What happened was for a time (one closing and reopening of the browser) the work tabs were fine, then the tabs went from hibernation to active, while I was looking at a site, layered tabs became single tabs, all tabs in recent closed tabs and recent closed windows list was gone. All had to be restored, only to have it happen again. After installing an older version of Firefox, even though the single window mode was not enabled and “show my windows and tabs from last time”, I was getting my home page and my work tabs in one window all active. I finally got that fixed. But, the Tab Utilities and Firefox 26 always gets the above described result.

I am currently trying the newer 1.5.20 with an older version of Firefox, and I am waiting to see if there is any problems; hopefully there is none.

I, also, checked for viruses, and there was none.  
Firefox, when I e-mailed them about the problem, stated that the blame was fully the add-on's fault (I do not fully believe this). They also stated on having Firefox remove the add-on from their list. This I do not want to happen as I told them. 

Tab Utilities is a great research tool, and works (at least 1.5.1, on the newer one we will see) fine with older versions of Firefox.  

However, there is a serious problem that I believe is a compatibility issue; even though when Firefox installs their update they check for compatibility of add-ons and disables any incompatible add-ons. This needs to be addressed and fixed.

Thank you,
Mr R. Miller

**Comment by ithinc** [comment]
Probably it is a conflict with some other extension. TU 1.5.20 alone on Fx26 works just fine.

**Comment by ithinc** [comment]
See #30.

**Comment by R-Miller** [comment]
I thought I should tell you that the problems between Tab Utilities and Mozilla Firefox 27 were solved.
A
 cleaning program that I have had a glitch where it checks for updates 
(I had closed this option still it did it), though I had not requested 
Tab Utilities to be updated, it did it . Now have 1.5.27.
This 
version, though in the past was not compatible with Mozilla Firefox 27 
(the problems were discussed in past e-mails), is now compatible with 
Firefox 27, and all problems seem to have disappeared (I have tried it 
for several days with none of the problems of the past). I do not know 
how, or what, happened only that it did.
Thank you, 

Mr. Miller

On Wednesday, February 26, 2014 5:08 PM, ithinc notifications@github.com wrote:

See #30.
